validation/StandardHypoTestDemo.py

```diff
@@ -0,0 +1,74 @@
+import ROOT
+import numpy as np
+
+
+def StandardHypoTestDemo(
+    infile,
+    ntoys,
+    workspaceName="combined",
+    modelSBName="ModelConfig",
+    dataName="obsData",
+):
+
+    file = ROOT.TFile.Open(infile)
+    w = file.Get(workspaceName)
+    sbModel = w.obj(modelSBName)
+    data = w.data(dataName)
+
+    bModel = sbModel.Clone()
+    bModel.SetName(modelSBName + "B_only")
+    var = bModel.GetParametersOfInterest().first()
+    oldval = var.getVal()
+    var.setVal(0)
+    bModel.SetSnapshot(ROOT.RooArgSet(var))
+    var.setVal(oldval)
+
+    var = sbModel.GetParametersOfInterest().first()
+    sbModel.SetSnapshot(ROOT.RooArgSet(var))
+
+    profll = ROOT.RooStats.ProfileLikelihoodTestStat(bModel.GetPdf())
+    profll.SetOneSidedDiscovery(False)
+    profll.SetOneSided(True)
+    hypoCalc = ROOT.RooStats.FrequentistCalculator(data, bModel, sbModel)
+    # profll.SetPrintLevel(2)
```

Comment:
This is fine, but at this level why not make it a `Click` CLI script?

Reply:
b/c my ROOT image doesn't have click installed. my memory is a bit vague: did we bulid a pyhf-root validation image?

Reply:
Yup. https://hub.docker.com/r/pyhf/pyhf-validation/tags But it needs to be updated.